### PR TITLE
Display potion amplification level

### PR DESCRIPTION
**Pull request by ewewukek**
hi! I believe this would be a useful feature :3
amplification_level

### CODE REVIEW

File: src/main/java/se/icus/mag/statuseffecttimer/mixin/StatusEffectTimerMixin.java [hunk 1]
```diff
@@ -55,8 +56,12 @@ private void renderDurationOverlay(MatrixStack matrices, CallbackInfo c) {
 					}
 
 					String duration = getDurationAsString(statusEffectInstance);
-					int length = client.textRenderer.getWidth(duration);
-					drawStringWithShadow(matrices, client.textRenderer, duration, x + 13 - (length / 2), y + 14, 0x99FFFFFF);
+					int durationLength = client.textRenderer.getWidth(duration);
+					drawStringWithShadow(matrices, client.textRenderer, duration, x + 13 - (durationLength / 2), y + 14, 0x99FFFFFF);
```
Comment (magicus) [inline src/main/java/se/icus/mag/statuseffecttimer/mixin/StatusEffectTimerMixin.java:60]: Do you think you could extract the `0x99FFFFFF` color code into a constant? It was ugly as it was to have it hard-coded like that, and duplicating it makes it worse. :-)

**Comment by magicus** [comment]
Hi @ewewukek,

Thank you for you contribution! This is a great idea.

**Comment by magicus** [comment]
Gah, I really need to fix that CI workflow. :)

**Comment by ewewukek** [comment]
I just found out mixin lib doesn't allow public fields in mixin classes. sorry didn't tested it earlier
```
diff --git a/src/main/java/se/icus/mag/statuseffecttimer/mixin/StatusEffectTimerMixin.java b/src/main/java/se/icus/mag/statuseffecttimer/mixin/StatusEffectTimerMixin.java
index 6d75519..9e48209 100644
--- a/src/main/java/se/icus/mag/statuseffecttimer/mixin/StatusEffectTimerMixin.java
+++ b/src/main/java/se/icus/mag/statuseffecttimer/mixin/StatusEffectTimerMixin.java
@@ -24,7 +24,7 @@ import java.util.Collection;
 @Environment(EnvType.CLIENT)
 @Mixin(InGameHud.class)
 public abstract class StatusEffectTimerMixin extends DrawableHelper {
-	public static final int TEXT_COLOR = 0x99FFFFFF;
+	private static final int TEXT_COLOR = 0x99FFFFFF;
 
 	@Shadow @Final
 	private MinecraftClient client;
```

**Comment by magicus** [comment]
Oh. :-( Something was nagging me about this in the back of my mind, but I assumed that it worked. *sigh*

**Comment by magicus** [comment]
I fixed it. Sorry for the trouble; your original patch was fine and I should have just kept my mouth shut. :)

**Pull request merged**